ckanext/canada/templates/internal/user/edit_user_form.html

```diff
@@ -39,17 +39,17 @@
   </fieldset>
 
   <div class="form-actions">
+    {{ form.required_message() }}
```

Comment:
why is a button a required field?

Reply:
or you're just moving the location of this text up in the form?

Reply:
@wardi Yeah just moving it up as it was in between buttons before.